package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@terrestris/ol-util",
-  "version": "7.0.0",
+  "version": "7.1.0",
```

Comment:
Is this change intended?

Reply:
no, somehow the commit from the last release made it into this MR. fixed via rebase